tests/datatypes/test_str.py

```diff
@@ -609,6 +609,22 @@ def test_strip(self):
 
             print('done.')
             """)
+        
+    def test_capitalize_no_args(self):
+        self.assertCodeExecution("""
+        x = 'abcde'
```

Comment:
@freakboy3742 Thank you, I have a little experience in tdd. Now I tried to make test more complete.